utility/src/test/java/org/oppia/android/util/profile/NameValidatorTest.kt

```diff
@@ -0,0 +1,26 @@
+package org.oppia.android.util.profile
+
+import com.google.common.truth.Truth
+import org.junit.Test
+
+class NameValidatorTest {
+
+  private val allowedNamesList = listOf<String>("जिष्णु", "Ben-Henning", "Rajat.T", "جيشنو")
+
+  private val unAllowedNamesList =
+    listOf<String>("जिष्णु7", "Ben_Henning", "Rajat..T", "جيشنو^&&", " ", ".", "Ben Henning")
+
+  @Test
+  fun testNameValidator_addUnAllowedName_returnFalse() {
+    unAllowedNamesList.forEach {
+      Truth.assertThat(!NameValidator.isNameValid(it))
+    }
+  }
+
+  @Test
+  fun testNameValidator_addAllowedName_returnTrue() {
+    allowedNamesList.forEach {
+      Truth.assertThat(NameValidator.isNameValid(it))
```

Comment:
You're not asserting anything. This should be: assertThat(value).isEqualTo(expectedValue), though because you're testing a boolean you can just use 'isTrue/isFalse'. See  other assertThat() calls in the codebase for reference.

You should also directly import assertThat (per convention & best practices).